docs/index.html

```diff
@@ -85,7 +85,9 @@ <h1 class="desc-h1">Nighthawk and Meshery</h1>
                     three choices of load generator for
                     characterizing and managing the performance
                     of service meshes and their workloads.</p> -->
-                  
+                    <div class="star-meshery"><a class="star-meshery-button" href="https://github.com/meshery/meshery" target="_blank" 
+                "> ⭐️ Star Meshery Repository </a>
+                    </div>
```

Comment:
@vednp Thanks for the PR, let's not create a button like this. 
Please refer to the https://meshery.io/ site, we need to replicate it. Also, ensure to add this star to the repository button in the first section (under the hero image).

Reply:
Sure, Is the position correct or should I add it in the hero section like in meshery's website?